kotlinx-coroutines-core/js/test/TestBase.kt

```diff
@@ -10,9 +10,11 @@ public actual val isStressTest: Boolean = false
 public actual val stressTestMultiplier: Int = 1
 
 public actual open class TestBase actual constructor() {
+    public actual val isJs = true
```

Comment:
I strongly dislike the name of this flag.

Consider the following scenario: we learn that there's no longer a two-second timeout on JS (also, where does the timeout come from?). However, it doesn't give us a clear indication to change anything about these tests: `isJs` should still be true. Instead, we're supposed to go looking through the tests to find those that are affected by the timeout no longer being there.

As an alternative, there's a much worse-sounding but, in my opinion, the more accurate name `timesOutAfterTwoSeconds` or something like this. This is what distinguishes the JS platform currently, this is what tests check for, the documentation for it is the place that could explain where the timeout comes from.

Reply:
>where does the timeout come from?

The underlying limitation of the Mocha test framework.

The rationale of having `js` in the name is mostly due to the fact that JS is tremendously slow for this kind of tests.
Renamed to `boundByJsTestTimeout` which seems to be a good middle-ground

Reply:
This is a good name, I agree.